annotations/3.1/_index.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "Jakarta Annotations 3.1 (Under development)"
+date: 2025-04-21
+summary: "Release for Jakarta EE 12"
+---
+Jakarta Annotations defines a collection of annotations representing common semantic concepts that enable
+a declarative style of programming that applies across a variety of Java technologies.
+
+The purpose of this release is to introduce annotations for running workloads on virtual threads, such as @RunOnVirtualThread,
```

Comment:
I think it is better to create an issue for a discussion on whether this is required first rather than mandate the necessity of even introducing an annotation.

Reply:
This spec is to provide some common APIs to be used by other specs. Which specs ask for this annotation?